### ISSUE TITLE
Dropdown menu (like the standard entities card does with an input_select)

### ISSUE THREAD

**Issue by jimz011** [custom-cards/button-card#224]
**Is your feature request related to a problem? Please describe.**
I'd wish to use buttons that only show an icon (like this card does exactly) in conjunction with an input_select. In standard lovelace with the core entities card you will get a dropdown menu. However that card can not be modified and it doesn't use the great 1/1 aspect_ratio that this card can do.

**Describe the solution you'd like**
I'd love to get a dropdown menu with the input_select options when I do a tap_action (or any other action that can trigger this)

**Describe alternatives you've considered**
I have tried using the core card and style it, but I can't set a dynamic height. It looks bad on a smartphone. There are no other cards that are alike as far as I know but the core card. 

**Additional context**
As always your card is great thanks.

**Comment by RomRider** [comment]
Did you try to embed the input_select in a custom field?

**Comment by jimz011** [comment]
Hey, long time no see @RomRider I hope you are doing well.

Anyways, yes I believe I did try (long time ago though). But it makes the entire card show up inside a button card. What I had wished for was a button like any other button but when pressed instead of navigating or service call it would show an input_select dropdown menu.

Example: when pressing the menu icon (this is a button-card) it will show a dropdown menu.

![8028A704-C71E-4D41-B6A1-EB336808F32F](https://user-images.githubusercontent.com/47046878/71920000-34d56380-3186-11ea-9cbb-22d9d1a082cf.jpeg)

I understand if you don’t feel this is a good addition or it might simply be not possible or too hard to implement. But seeing no other card does this except the core entities I think it might be nice to have. Also the official entities card with input_select is extremely hard to style. As I believe it wasn’t meant to be used like that.

Once again thanks for all your work! We have missed you haha!

Jimz011

**Comment by iantrich** [comment]
I don't see how this would fall within the realm of the button-card. It sounds like you're wanting an element for input_select, more so, which could be a nice addition to core or a custom element, but not the button-card itself

**Comment by jimz011** [comment]
Yes and no, I wish for button-card as it is the best looking card out there. And yes I actually want the input_select element to be opened when pressing on the button.

Like I said I understand that this might seem to not be handy. But input_select dropdown menus can only be called with an entities card (not even with a core button) which is too bad. But it is alright, I have created a view which essentially does the same. But having a dropdown menu open on a button press would be amazing. It would remove the entire need for a separate view as I can automate the input select to open a view instead.

Anyways thanks for the answers. I wish I could show an actual example and usecase for it but my editing skills are not that great.

If I can find a way to create a screenshot showing this I will post it. In any other case you can simply close this issue if there is no interest. Srry for the inconvenience.

**Comment by coolguymatt** [comment]
Did you ever figure out how to do this?

**Comment by johntdyer** [comment]
+1
